mlrun/model_monitoring/db/tsdb/v3io/v3io_connector.py

```diff
@@ -504,19 +504,29 @@ def read_metrics_data(
         if type == "metrics":
             table_path = self.tables[mm_schemas.V3IOTSDBTables.METRICS]
             name = mm_schemas.MetricData.METRIC_NAME
+            columns = [mm_schemas.MetricData.METRIC_VALUE]
             df_handler = self.df_to_metrics_values
         elif type == "results":
             table_path = self.tables[mm_schemas.V3IOTSDBTables.APP_RESULTS]
             name = mm_schemas.ResultData.RESULT_NAME
+            columns = [
+                mm_schemas.ResultData.RESULT_VALUE,
+                mm_schemas.ResultData.RESULT_STATUS,
+                mm_schemas.ResultData.RESULT_KIND,
+            ]
             df_handler = self.df_to_results_values
         else:
             raise ValueError(f"Invalid {type = }")
 
+        groupby = [name, mm_schemas.WriterEvent.APPLICATION_NAME]
+
```

Comment:
Why you need to group by in this case? You need all the data no?

Reply:
You are right. It's not needed, I removed it.